gap/attr.gi

```diff
@@ -384,14 +384,10 @@ InstallMethod(OutNeighbours, "for a digraph",
 [IsDigraph],
 function(digraph)
   local out;
-  if IsBound(digraph!.adj) then
-    return digraph!.adj;
```

Comment:
Why is this change included here? And what is the explanation for its deletion?

Reply:
Didn't we remove the component `adj` from all digraphs at some point in the not too distant past? This was originally used to hold the `OutNeighbours` for access in the C code, but if I recall this was unnecessary, and I think you removed it. I believe this code is a remnant of the older code, and I noticed it when writing tests for `MaximalAntiSymmetricSubdigraph` (it was not covered by any tests, and removing it seems to have no effect). Perhaps it should be in another PR, but it is in a separate commit at least.

Reply:
Ah right, no problem. I'm not sure if I did the changes, but yes this the `adj` component is removed, and certainly if these lines are not being covered by tests then it's not needed. I'm not normally against a PR changing unrelated things, but my default assumption when reviewing a PR is that the changes are related, and so I was confused at this point because I couldn't work out what the relation was here.